src/ClientData/CallstackDataTest.cpp

```diff
@@ -200,4 +200,144 @@ TEST(CallstackData, FilterCallstackEventsBasedOnMajorityStartExcludesFunctionToS
                                                                          event4, event5, event6}));
 }
 
+TEST(CallstackData, ForEachCallstackEventOfTidInTimeRangeDiscretized) {
+  CallstackData callstack_data;
+
+  constexpr uint32_t kTid = 42;
+  constexpr uint32_t kAnotherTid = 43;
+
+  constexpr uint64_t kCloneAddress = 0x10;
```

Comment:
I find it very strange to use a name like "kCloneAddress" when this test is just about computing timestamps and pixels and has nothing to do with the actual content of the callstacks.